tests/testthat/_snaps/dots-ellipsis.md

```diff
@@ -0,0 +1,38 @@
+# error if dots not used by another function
+
+    Code
+      expect_error(f(x = 10, c = 3), class = "rlib_error_dots_unused")
+
+# error if dots named
+
+    Code
+      (expect_error(f(1, 2, 3, xy = 4, x = 5), class = "rlib_error_dots_named"))
+    Output
+      <error/rlib_error_dots_named>
+      2 arguments in `...` had unexpected names.
+      x We detected these problematic arguments:
+      * `xy`
+      * `x`
```

Comment:
Have we ever discussed indenting these sub bullets? I think that might look nice

Reply:
Ah there is an ongoing Slack discussion with @gaborcsardi and @jennybc about this on Slack. We might add and `indent` argument to `format_error_bullets()` and cli might gain special interpolation syntax for this.